modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/rest/types.ts

```diff
@@ -54,7 +55,27 @@ export type UserGroup = {
 	name: string;
 };
 
+export type UserActions = {
+	'delete-user-account': {
+		href: string;
+		method: string;
```

Comment:
This code is repeated many times bellow, it could be a type;

```
    'delete-user-account': ActionPermissionProperties
```

for example

Reply:
Done!